frontend/templates/troubleshooting/index.tsx

```diff
@@ -46,6 +57,51 @@ const Wiki: NextPageWithLayout<{
    );
 };
 
+function NavBar() {
+   return (
+      <Flex
+         w="100%"
```

Comment:
I am not sure how much styling tweaking you are planning on doing for the early stages of Vulcan, but just to note, without some padding on the children elements. The breadcrumbs and area link text will ride the borders.

![image](https://user-images.githubusercontent.com/22064420/219483990-803c1263-770e-45d6-8f48-8090b416feb2.png)

Reply:
I'm fine with that right now, because they're empty. When I go to add stuff to them, I'll either need to add a `gap` on this flexbox or add padding to them.